### PR TITLE
[NFC] Clang-format fixes.

### PR DESCRIPTION
**Pull request by maarquitos14**
Most likely the clang-format checker was down when https://github.com/intel/llvm/pull/14575 ran CI, so a few changes non-clang-format-compliant made it in. Applying fixes to those.

### CODE REVIEW

**Comment by maarquitos14** [comment]
@aelovikov-intel FYI

File: marcos.patch [hunk 1]
```diff
@@ -0,0 +1,92 @@
+From 2b6e88d4d01fea816f525fafc71676603f7a8c53 Mon Sep 17 00:00:00 2001
```
Comment (mdtoguchi) [inline marcos.patch:1]: Looks like this patch file snuck into your commit

Comment (maarquitos14) [reply marcos.patch:1]: Oops, thanks for catching this!

Comment (maarquitos14) [reply marcos.patch]: Fixed :)

**Review by steffenlarsen — APPROVED** [review]

**Comment by maarquitos14** [comment]
@bader Friendly ping.

**Comment by maarquitos14** [comment]
@mdtoguchi F
riendly ping.

**Review by bader — APPROVED** [review]

**Review by mdtoguchi — APPROVED** [review]

**Comment by maarquitos14** [comment]
@intel/llvm-gatekeepers this is ready to merge.

**Pull request merged**